docs/project/contributing.md

```diff
@@ -26,7 +26,7 @@ Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint]
 - [Git](https://github.com/git-guides/install-git)
 
 ## Contributing flow
-<img src="\img\cheatsheet.jpg" width="350"/> 
+<img src="\img\cheatsheet.jpg" max-width/> 
```

Comment:
lol, 

The rigth should be `max-width="350"`

Reply:
> 

I know, I thought the same thing. But when I used "npm start" it didn't generate any difference between "max-wmax-width="350" and "max-width", but I'll put it anyway.

Reply:
The difference is in the mobile not desktop :)

Reply:
Living and learning!

Reply:
Living and learning!
It's done. 
Tks! :D